datafusion/sqllogictest/test_files/arrow_files.slt

```diff
@@ -64,6 +72,13 @@ SELECT * FROM arrow_partitioned ORDER BY f0;
 3 baz true 456
 4 NULL NULL 456
 
+# dynamic select arrow file in the folder
+query ITB
+SELECT * FROM '../core/tests/data/partitioned_table_arrow/part=123' ORDER BY f0;
```

Comment:
Dynamic query doesn't support the partitioned folder. It could be an enhancement issue for it.